server/fishtest/templates/machines_table.mak

```diff
@@ -20,7 +20,7 @@
           gcc_version = ".".join([str(m) for m in machine['gcc_version']])
           compiler = machine.get('compiler', 'g++')
           python_version = ".".join([str(m) for m in machine['python_version']])
-          version = machine['version']
+          version = str(machine['version'])+("+" if machine['modified'] else "")
```

Comment:
Black adds spaces around the plus sign:
```python
version = str(machine['version']) + ("+" if machine['modified'] else "")
```
Other options are still ugly for the flood of single/double quotes, parentheses/brackets/braces, the last one is funny :)
```python
version = "".join([str(machine['version']), "+" if machine['modified'] else ""])
version = f"{machine['version']}{'+' if machine['modified'] else ''}"
version = "{}{}".format(machine['version'], "+" if machine['modified'] else "")
version = str(machine['version']) + '+' * machine['modified']
```

Reply:
The last one is short but it is a bit of a hack (it depends on the fact that Python interprets True as 1 and False a 0). But I guess it is clear what it is meant. I will take it.

Reply:
Done (not tested since this laptop doesn't have fishtest).

Reply:
DEV updated, it works.